ui/perfherder/helpers.js

```diff
@@ -754,3 +759,6 @@ export const nudgeAlert = (dataPoint, towardsDataPoint) => {
   const alertId = dataPoint.alert.id;
   return update(getApiUrl(`/performance/alert/${alertId}/`), towardsDataPoint);
 };
+
+export const getRevisionUrl = (revision, projectName) =>
```

Comment:
I think this is general enough to belong in `ui/helpers/url.js` rather than a perfherder-only helper?